### PR TITLE
Add message on samples page and redesign empty cart page

### PR DESCRIPTION
**Pull request by nikunjsdev**
Fix samples page when there are no samples, redesign empty cart page and redesign buttons throughout the site.

empty-card

no-samples

### CODE REVIEW

File: src/views/Cart/scss/index.scss [hunk 1]
```diff
@@ -76,20 +76,50 @@
...
     &__action {
       text-align: center;
       margin-top: $spacer;
     }
+    &-icon {
+      background-size: cover;
+      background-repeat: no-repeat;
+      width: 262px;
+      height: 190px;
+      margin-bottom: 45px;
```
Comment (zshanhui) [inline src/views/Cart/scss/index.scss]: Use `$spacer` or `rem` units for margin, paddings.

Will just comment this one, but fix the others too.

File: src/views/CollectionIndex/scss/index.scss [hunk 2]
```diff
@@ -38,7 +38,7 @@
   background: $light-grayish-orange;
   padding: 16px 35px;
   font-size: 14px;
-  border-radius: 7px;
+  border-radius: 25px;
   box-shadow: 0px 3px 6px #00000029;
```
Comment (zshanhui) [inline src/views/CollectionIndex/scss/index.scss]: You can use a variable for this whole expression `0px 3px 6px #00000029`

Remember to change the `box-shadow` below too.

File: src/views/CollectionIndex/Samples/Page.tsx [hunk 3]
```diff
@@ -49,49 +50,58 @@ export const Page = ({ data, cart, collectionId, history }: IPage) => {
...
-                    />
-                  </div>
-                  <div className="wrapper-img-main-inner--price">
-                    <span className="old-price">
-                      <del>${pricing.priceRange.start.gross.amount}</del>
-                    </span>
-                    <span className="new-price">
-                      ${pricing.priceRange.start.net.amount}
-                    </span>
+          {sampleCount > 0 ? (
```
Comment (zshanhui) [inline src/views/CollectionIndex/Samples/Page.tsx]: You can rename `sampleCount` to `samplesExist`, and just use `samplesExist ? (...) : (...)`

File: src/views/CollectionIndex/Samples/Page.tsx [hunk 4]
```diff
@@ -12,6 +12,7 @@ interface IPage {
 export const Page = ({ data, cart, collectionId, history }: IPage) => {
   const [samples, selectedSample] = useState([]);
   const collect = [];
+  const sampleCount = data.products.totalCount;
```
Comment (zshanhui) [inline src/views/CollectionIndex/Samples/Page.tsx]: Rename to `samplesExist`

File: src/views/CollectionIndex/Samples/Page.tsx [hunk 5]
```diff
@@ -49,49 +50,58 @@ export const Page = ({ data, cart, collectionId, history }: IPage) => {
...
+                        key={idx}
+                      />
+                    </div>
+                    <div className="wrapper-img-main-inner--price">
+                      <span className="old-price">
+                        <del>${pricing.priceRange.start.gross.amount}</del>
+                      </span>
+                      <span className="new-price">
+                        ${pricing.priceRange.start.net.amount}
+                      </span>
```
Comment (zshanhui) [inline src/views/CollectionIndex/Samples/Page.tsx]: Sample as other PR, if `gross.amount` is equal to `net.amount`, only need to show `net.amount`

**Review by zshanhui — CHANGES_REQUESTED** [review]
@nikunjsdev Hey, all done with the review. Only a few changes, then we're ready to merge. Good job!

File: src/views/Cart/scss/index.scss (hunk 1 above)
Comment (nikunjsdev) [reply src/views/Cart/scss/index.scss]: Sorry didn't your last statement

File: src/views/CollectionIndex/Samples/Page.tsx (hunk 3 above)
Comment (nikunjsdev) [reply src/views/CollectionIndex/Samples/Page.tsx]: Done

File: src/views/CollectionIndex/scss/index.scss (hunk 2 above)
Comment (nikunjsdev) [reply src/views/CollectionIndex/scss/index.scss]: Done

File: src/views/CollectionIndex/Samples/Page.tsx (hunk 4 above)
Comment (nikunjsdev) [reply src/views/CollectionIndex/Samples/Page.tsx]: Done

File: src/views/CollectionIndex/Samples/Page.tsx (hunk 5 above)
Comment (nikunjsdev) [reply src/views/CollectionIndex/Samples/Page.tsx]: Done

**Review by zshanhui — APPROVED** [review]
@nikunjsdev LGTM!

**Pull request merged**